scripts/trivia.js

```diff
@@ -784,11 +785,16 @@ TriviaGame.prototype.finalizeAnswers = function () {
                         if (this.triviaPlayers[id2].name === name) break;
                     }
                     if(trivData.eventFlag){
-                        eventElimPlayers.push(this.triviaPlayers[id2].name);
+                        sortArray.push(this.triviaPlayers[id2].name);
                     }
                     this.unjoin(id2);
                 }
             }
+            //Add players to sort array as eliminated, then randomize listing so that it isn't based on join order when ties happen.
+            sortArray.sort(function(a, b){return 0.5 - Math.random()});
```

Comment:
Also, I need a way to isolate the new additions to the eventElimPlayers array (the one safari uses) without shuffling players eliminated in previous rounds.